<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>miz7035_demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" rowspan="2" colspan="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Transfer Test</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>C2H</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="comboBox_c2h">
            <item>
             <property name="text">
              <string>1K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>64K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>128K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>512K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8M</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pushButton_c2h_start">
            <property name="text">
             <string>START</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="lineEdit_c2h"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>H2C</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="comboBox_h2c">
            <item>
             <property name="text">
              <string>1K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>64K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>128K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>512K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8M</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="pushButton_h2c_start">
            <property name="text">
             <string>START</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="lineEdit_h2c"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>User Reg</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ADDR</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_ureg_addr"/>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>DATA</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="lineEdit_ureg_data"/>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_ureg_get">
            <property name="text">
             <string>GET</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>58</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="pushButton_ureg_set">
            <property name="text">
             <string>SET</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEdit_reso">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_sel_dir">
            <property name="text">
             <string>Select Pic</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="pushButton_play">
            <property name="text">
             <string>PLAY</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2" colspan="2">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="lineEdit_satu"/>
          </item>
          <item row="2" column="2" colspan="2">
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>258</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_c2h</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>combo_box_c2h_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1060</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>1095</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_h2c</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>combo_box_h2c_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1098</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>1131</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_c2h_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push_button_c2h_start_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1172</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>1239</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_h2c_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push_button_h2c_start_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1192</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>1222</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ureg_get</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push_button_ureg_get_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1101</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>1157</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ureg_set</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push_button_ureg_set_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1242</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>1253</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_sel_dir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push_button_sel_dir_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1124</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>1152</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_play</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>push_button_play_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1225</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>1163</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>combo_box_c2h_slot(int)</slot>
  <slot>combo_box_h2c_slot(int)</slot>
  <slot>push_button_c2h_start_slot()</slot>
  <slot>push_button_h2c_start_slot()</slot>
  <slot>push_button_ureg_get_slot()</slot>
  <slot>push_button_ureg_set_slot()</slot>
  <slot>check_box_repeat_slot(bool)</slot>
  <slot>push_button_sel_dir_slot()</slot>
  <slot>push_button_play_slot()</slot>
 </slots>
</ui>
